provision-proctor/setup.sh

```diff
@@ -117,6 +97,9 @@ if [[ -z "$proctorNumber" ]]; then
     proctorNumber="$(randomChar;randomChar;randomChar;randomNum;)"
 fi
 
+declare proctorName="monitoring$(randomChar;randomChar;randomChar;randomNum;)"
+declare proctorNumber="$(randomNum;randomNum;randomNum;randomNum;)"
+
```

Comment:
Line 120 echos `$teamName` but it isn't declared yet.
Line 179 sets kvstore for `$teamName`

Reply:
Removed all occurrences of teamName from the script.